test/graphql/types/userList.spec.ts

```
@@ -0,0 +1,45 @@
+import { createTestClient } from 'apollo-server-testing';
+import { ApolloServer } from 'apollo-server';
+import { schema } from '~/src/graphql/schema';
+import { context } from '~/src/graphql/context'; // Ensure 'context' is correctly exported from this module
+
+describe('userList', () => {
+    let query: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)